### PR TITLE
WIP: Parser for 'sos_commands/foreman/foreman-debug/var/log/candlepin/error.log'

### PR DESCRIPTION
**Pull request by sagaraivale**
@vishwanathjadhav @xiangce  : 
Kindly review the parser for 'sos_commands/foreman/foreman-debug/var/log/candlepin/error.log'

### CODE REVIEW

**Comment by jhjaggars** [comment]
Hey @sagaraivale please rebase this PR against master and push it up again.  It's very hard to tell what's going on with all the merging on top of your change.

File: insights/parsers/foreman_log.py [hunk 1]
```diff
@@ -38,3 +38,39 @@ class ProductionLog(LogFileOutput):
 class CandlepinLog(LogFileOutput):
     """Class for parsing ``candlepin/candlepin.log`` file."""
     pass
+
+
+@parser('candlepin_error_log')
+class Candlepin_Error_Log(LogFileOutput):
```
Comment (bfahr) [inline insights/parsers/foreman_log.py]: @sagaraivale class names should be camel-case with no underscores.

File: insights/config/specs.py [hunk 2]
```diff
@@ -20,6 +20,7 @@
                                     SimpleFileSpec("sos_commands/filesys/blkid")]),
     "bond"                      : PatternSpec(r"proc/net/bonding/bond.*"),
     "brctl_show"                : CommandSpec("/usr/sbin/brctl show"),
+    "candlepin_error_log"       : SimpleFileSpec(r"sos_commands/foreman/foreman-debug/var/log/candlepin/error.log", large_content=True),
     "candlepin.log"             : First([SimpleFileSpec(r"var/log/candlepin/candlepin.log", large_content=True),
```
Comment (bfahr) [inline insights/config/specs.py]: Please move the new line below `candlepin.log`.

**Comment by xiangce** [comment]
@sagaraivale this PR seems the same to #476, do you want to close this one?

**Comment by sagaraivale** [comment]
@xiangce : Yes, I want to close this PR request #467

**Comment by vishwanathjadhav** [comment]
@sagar I think you want to close #476, confirm it first and close it by
your self.

**Pull request closed without merging**